immobilus/logic.py

```diff
@@ -1,31 +1,59 @@
+"""Main library module to prepare the target `immobilus` class."""
+import calendar
+import copyreg
 import sys
 import time
-import calendar
 from asyncio import iscoroutinefunction
-from datetime import datetime, date, timedelta, tzinfo
+from datetime import date, datetime, timedelta, tzinfo
 from functools import wraps
 
 from dateutil import parser
 
-try:
-    import copy_reg as copyreg
-except ImportError:
-    import copyreg
 
-TIME_TO_FREEZE = None
-TZ_OFFSET = 0
+class Config(object):
```

Comment:
Looks like it's not a Config, since values are not changeable outside. Mb smth like TemporaryData or smth similar?